src/util/filterIncludes/index.js

```diff
@@ -12,9 +12,10 @@ import { error } from '../'
  */
 export async function filterIncludes (included, { id, type }) {
   try {
-    return included.filter(el => {
+    const filtered = included.filter(el => {
       return el.id === id && el.type === type
     })[0] || { id, type }
+    return {...filtered}
```

Comment:
This spread operator looks useless and adds an extra runtime dependency as only recent node and browser (last year) versions support it.

Runtime dependency increases the total import cost by `0.3kb` and over the set limit.